index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Page Title</title>
+<meta charset="UTF-8">
+<link rel="stylesheet" href="styles.css">
+<div>
+  <header id="header">
+    <div class="logo">
```

Comment:
- [ ] kindly suggest try not to use id as possible. Id have a lot of problem in working environment. It is very specific and an id within an HTML page must be unique so the rule will apply for only one element. Please try using different classes.

Reply:
[OPTIONAL]
-[ ] Since this one is not in the project requirements, you may consider putting it with the flag [OPTIONAL] :smile:

Reply:
Thank you, Sir. Now I know when it's time to flag. 🤩